pr-deploy/src/app.ts

```diff
@@ -47,26 +48,34 @@ function initializeCheck(app: Application) {
 function initializeRouter(app: Application) {
   const router: IRouter<void> = app.route('/v0/pr-deploy');
   router.use(express.json());
-  router.post('/owners/:owner/repos/:repo/headshas/:headSha',
+  router.post('/owners/:owner/repos/:repo/headshas/:headSha/:exitCode',
     async(request, response) => {
       const github = await app.auth(Number(process.env.INSTALLATION_ID));
-      const {headSha, owner, repo} = request.params;
+      const {headSha, owner, repo, exitCode} = request.params;
       const pr = new PullRequest(github, headSha, owner, repo);
-      await pr.enableDeploymentCheck();
-      response.send({status: 200});
+
+      if (exitCode != 0) {
+        await pr.errorCompilationCheck();
+        response.send({status: 500});
```

Comment:
Why respond with 500? The action here succeeded, it failed somewhere else and that elsewhere knows it already :)

Reply:
True true